packages/client/hmi-client/src/components/drilldown/tera-drilldown-section.vue

```diff
@@ -40,5 +40,6 @@ main {
 	flex-grow: 1;
 	overflow-y: auto;
 	gap: 1.5rem;
+	padding: 1.5rem 1.5rem 1.5rem 1rem;
```

Comment:
We have CSS variables for that `--gap`.
See: https://github.com/DARPA-ASKEM/terarium/blob/main/packages/client/hmi-client/src/assets/css/theme/_variables.scss#L949-L951

Reply:
will do, but this has been reverted from the above comment